### DISCUSSION TITLE
[documentation]  [help-wanted] How to deploy same settings, buckets, app installs on another computer? [Bug]

### DISCUSSION THREAD

**Discussion by rdthree** [ScoopInstaller/Scoop#4752] [Help and Questions] [answered]
What is the most direct way to re-deploy all my installed scoop apps, buckets and settings on a fresh computer?

The .config doesn't seem to have that info, and I'm unsure how to import the information from the list of installed apps or get bucket locations from some of the custom buckets.  Thanks

**Comment by rashil2000** [comment]
- Install scoop on new computer.
- Copy ~/scoop/persist folder to new computer
- Install all your apps (you can list them by doing `scoop export`). Import command is not implemented yet.

**Comment by rashil2000** [comment, marked as answer]
Import command is currently in review: https://github.com/ScoopInstaller/Scoop/pull/5014
